### PR TITLE
Expand usage documentation 

### PR DESCRIPTION
**Pull request by L1ghtmann**
What does this implement/fix? Explain your changes.
---------------------------------------------------
Creates Configuration.md and Packaging.md

Does this close any currently open issues?
------------------------------------------
N/A

Any relevant logs, error output, etc?
-------------------------------------
N/A

Any other comments?
-------------------
I tried my best to include relevant items, but if something is missing or undesired, just say the word!

Where has this been tested?
---------------------------
**Operating System:** …

**Platform:** …

**Target Platform:** …

**Toolchain Version:** …

**SDK Version:** …

### CODE REVIEW

File: docs/Packaging.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+- Sileodepiction:
+    - A link to your package's depiction (for newer package managers)
+
+Aside from a few jailbroken iOS-specific fields, all available fields can be found on https://www.debian.org/doc/debian-policy/ch-controlfields.html.
+
+## Maintainer scripts
+
+As with standard Debian packages, maintainer scripts can be added to your project to be run at various points throughout its (un)install cycle.
+
+The scripts should be placed in $THEOS_LAYOUR_DIRECTORY and include:
```
Comment (ExTBH) [inline docs/Packaging.md:86]: i think you meant `$THEOS_LAYOUT_DIRECTORY` ?

Comment (L1ghtmann) [reply docs/Packaging.md:86]: Yup! Thanks for catching that.

**Review by kirb — APPROVED** [review]
There's a few things I want to tweak, but it's good to merge as-is and I can do those later.

File: docs/Packaging.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+- .dylib
+- .a
+- .so
+- .deb
+- .ipa
+- null packages
+- .pkg
+- .rpm
+- .exe
+- .dll
```
Comment (kirb) [inline docs/Packaging.md:24]: How come executable formats are mixed with packaging formats here?

Comment (kirb) [inline docs/Packaging.md:24]: How come executable formats are mixed with packaging formats here?

**Comment by kirb** [comment]
Merged :+1: Would appreciate if you could look into that question I had when you can.

File: docs/Packaging.md (hunk 2 above)
Comment (L1ghtmann) [reply docs/Packaging.md:24]: I can't remember if there was a specific reason ... likely just got carried away and listed all of the file formats I came across. Whoops!

Comment (L1ghtmann) [reply docs/Packaging.md:24]: Removed non-packaging formats from the list in a recent commit to #19.

**Pull request closed without merging**